### ISSUE TITLE
Access denied to local storage

### ISSUE THREAD

**Issue by serpent7776** [reismahnic/FarmHorrorGame#1]
When I wanted to exit from settings and pressed ESC I got 
`failed to read the localstorage property from window. access denied for this document`
and game cannot continue and needs restart.

**Comment by reismahnic** [comment]
Oh no! Thanks for reporting this, serpent.
Can I ask what version of the game you were playing? PC, Mac or Web?

**Comment by serpent7776** [comment]
Was playing web version on chromium

**Comment by reismahnic** [comment]
I haven't replicated the bug on Firefox or Chrome so I'm gonna have to assume that's a Chromium thing :/
Thanks for letting us know though!

**Comment by serpent7776** [comment]
That might be some kind of setting in chromium that I have enabled, but not sure what exactly.

**Comment by reismahnic** [comment]
Yeah, also possible RPG Maker just hasn't been designed to run on Chromium well yet.
